keps/sig-node/127-user-namespaces/README.md

```diff
@@ -341,7 +448,86 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
-TBD
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+Most of the changes will be in a new file we will create, the userns manager.
+Tests are already written for that file.
+
+The rest of the changes are more about hooking it up so it is called.
+
+- `pkg/kubelet/kuberuntime/kuberuntime_container_linux.go`: 24.05.2022 - 90.8%
+- `pkg/volume/util/operationexecutor/operation_generator.go`: 24.05.2022 - 8.6%
```

Comment:
>> This seems super low.
>>
>> Can we first increase the coverage of this package before we start changing this?

> that seems out of scope to me. We are just plumbing some values when we call volumeMounter.SetUp() without adding any additional feature to the file

I will leave it up to SIG Node folks (@SergeyKanzhelev - please chime in), but I think this is the approach that is hurting us, because if we're only making this problem worse.
I understand that your changes here will be mostly mechanical, but it's the case for many changes. And we should use that opportunity to make the codebase better.